src/common/gui/CSurgeSlider.cpp

```diff
@@ -435,3 +435,32 @@ void CSurgeSlider::setBipolar(bool b)
 
    setDirty();
 }
+
+bool CSurgeSlider::onWheel(const VSTGUI::CPoint& where, const float &distance, const VSTGUI::CButtonState& buttons)
+{
+   double rate = 0.1 * moverate;
+   if (buttons & kRButton)
+      rate *= 0.05;
```

Comment:
Is it even possible to mouse wheel with the right button down? More curious than anything else